api/v1/swagger.yaml

```diff
@@ -3094,6 +3094,17 @@ paths:
             Filter by parentAbsolutePath; asterisk (*) wildcards ok.
           required: false
           type: string
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
```

Comment:
Since filtering for the hierarchy end point was done in the API layer, it made sense to store the tags the way it was created by the user.